sentry/api/sentry.api

```diff
@@ -1742,7 +1750,7 @@ public final class io/sentry/SentryTraceHeader {
 
 public final class io/sentry/SentryTracer : io/sentry/ITransaction {
 	public fun <init> (Lio/sentry/TransactionContext;Lio/sentry/IHub;)V
-	public fun <init> (Lio/sentry/TransactionContext;Lio/sentry/IHub;ZLio/sentry/TransactionFinishedCallback;)V
+	public fun <init> (Lio/sentry/TransactionContext;Lio/sentry/IHub;Lio/sentry/TransactionOptions;Lio/sentry/TransactionFinishedCallback;)V
```

Comment:
`m`: technically a breaking change, but I doubt anyone ever used that. Should we remove it altogether and just use the other ctor in the ActivityLifecycleIntegrationTest?

Reply:
I think that's fine as `SentryTracer` is `ApiStatus.Internal`